tests/unit/filing-template-mixin.spec.ts

```diff
@@ -96,6 +96,7 @@ describe('Registration Filing', () => {
     store.state.stateModel.registration.businessType = 'SP' // not DBA
 
     store.state.stateModel.registration.businessNumber = '111222333'
+    store.state.stateModel.registration.businessTypeConfirm = null
```

Comment:
So what value do we want to initialize this to...
- null, meaning it has no value (is neither true nor false)?
- false, since that's the default for the checkbox (ie, not checked)? (and is also the initial value in the store state object)

Another test case could check that businessTypeConfirm in the store gets a legitimate value (true or false) when loading a sparse registration filing.

Reply:
This has to be null if we use draftFiling.registration.businessTypeConfirm || null in the filing mixin, because the draft registration doesn't have the key businessTypeConfirm and the received value in the test run is always null. If we use draftFiling.registration.businessTypeConfirm (|| false), then we can change this to false.

Reply:
I changed it to false in my latest commit: c77ff41.